src/pds/ingress/client/pds_ingress_client.py

```diff
@@ -109,7 +111,10 @@ def perform_ingress(batched_ingress_paths, node_id, prefix, force_overwrite, api
         # Perform uploads to S3 in parallel based on number of files
         PARALLEL(delayed(ingress_file_to_s3)(ingress_response) for ingress_response in chain(*response_batches))
     except Exception as err:
-        logger.error("Ingress failed, reason: %s", str(err))
+        if isinstance(err, RequestException):
```

Comment:
The rules for exception handling (processed in-order) let you avoid the `isinstance` call:
```python
except RequestException as err:
    logger.error("Ingress failed, HTTP response text:\n%s", err.response.text)
except Exception as err:
    logger.error("Ingress failed, reason: %s", str(err))
```

Reply:
🤦 will fix